project/Setup.scala

```diff
@@ -0,0 +1,62 @@
+import sbt.Keys.javacOptions
+import sbt.TaskKey
+import sbtassembly.AssemblyKeys.assemblyMergeStrategy
+import sbtassembly.AssemblyPlugin.autoImport
+import sbtassembly.AssemblyPlugin.autoImport.{MergeStrategy, assembly}
+import sbtassembly.PathList
+
+/*
+ * Copyright 2024 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+object Setup {
+
+  //supported Scala versions
+//  val scala212 = Version.asSemVer("2.12.18") TODO
```

Comment:
oh okay these are still todo

Reply:
perhaps remove these and move the ones below into `Dependencies.scala` ?

Reply:
I want to use `SemVer` type from Absa commons. And no, supported version of Scala I wouldn't call dependency, but setup, so here on purpose.

Reply:
Ok, fair enough, I like it more